libexec/bootstrap/modules-v2/build-busybox.sh

```diff
@@ -59,7 +59,12 @@ echo "root:!:0:0:root:/root:/bin/sh" > "$SINGULARITY_ROOTFS/etc/passwd"
 echo " root:x:0:" > "$SINGULARITY_ROOTFS/etc/group"
 echo "127.0.0.1   localhost localhost.localdomain localhost4 localhost4.localdomain4" > "$SINGULARITY_ROOTFS/etc/hosts"
 
-curl "$MIRROR" > "$SINGULARITY_ROOTFS/bin/busybox"
+curl -f "$MIRROR" > "$SINGULARITY_ROOTFS/bin/busybox"
+
+if [ ! $? -eq 0 ]; then
```

Comment:
Agreed - I meant to do that last night but got distracted by this test failure (from the tests I added in this PR)